remc2/engine/Animation.cpp

```diff
@@ -51,7 +51,7 @@ void PlayInfoFmv(__int16 a1, __int16 a2, type_E17CC_0* a3x, char* path)//257160
 	x_WORD_D4004 = 0;
 	x_WORD_17DB58 = 0;
 	ActualKeyframe_17DB60 = 0;
-	x_DWORD_E12F4 = pdwScreenBuffer;
+	x_DWORD_E12F4x = (TColor*)pdwScreenBuffer;
```

Comment:
The pointer for the pdwScreenBuffer is actually a 1 byte index for a colour in the pallet. I am not sure TColor will not confuse people. TColorIdx maybe?